project/cmake/addons/addons/adsp.basic/adsp.basic.txt

```diff
@@ -1 +1 @@
-adsp.basic https://github.com/AchimTuran/adsp.basic master
+adsp.basic https://github.com/AchimTuran/adsp.basic fix_win32_cmake
```

Comment:
i would be very surprised if that differs on windows. @montellese you have done this with success right ?